cpp/daal/include/services/internal/daal_kernel_defines.h

```diff
@@ -35,6 +36,7 @@
 #define DAAL_KERNEL_SSE42
 #define DAAL_KERNEL_AVX2
 #define DAAL_KERNEL_AVX512
+#define DAAL_KERNEL_SVE
```

Comment:
The defines should be protected by an ifdef. No need for the x86 defines to exist in an Arm build, and vice versa